### PR TITLE
Get Started - bare metal org clean up

### PR DESCRIPTION
**Pull request by krisdale**
Clean up the Get Started/bare metal install pages/file organization so that they show up in the nav as intended (as sibling nav items, not nested children). This is followup to the new nav. 

Additionally, updated substitutions and attributions across pages under Get Started.

### CODE REVIEW

**Comment by mvincerx** [comment]
Minor minor change. After above commit is pushed, this is ready to roll. Thanks for all the work on this!

**Comment by krisdale** [comment]
@mvincerx Updated vmware player instructions to remove included images section (which contained the redundant attribution), and replace with a link to the supplemental info. Found a couple other minor edits in the virtual machine section which I included.

File: source/clear-linux/get-started/virtual-machine-install/vmw-player-preconf.rst [hunk 1]
```diff
@@ -61,14 +61,14 @@ Install the VMware Workstation Player hypervisor
 
 For additional help, see the `VMware Workstation Player guide`_.
 
-Clear Linux image types
-***********************
+|CL| image types
+****************
```
[the remark this answers is not in the document]
Comment (krisdale) [reply source/clear-linux/get-started/virtual-machine-install/vmw-player-preconf.rst]: @mvincerx Updated vmware player instructions to remove included images section (which contained the redundant attribution), and replace with a link to the supplemental info.

File: source/clear-linux/get-started/virtual-machine-install/vmware-esxi-install-cl.rst [hunk 2]
```diff
@@ -10,9 +10,6 @@ run |CL-ATTR| virtual machines in the cloud.
 This section shows you how to create a new :abbr:`VM (Virtual Machine)` and 
 manually install |CL| into it with VMware ESXi 6.5.
```
Comment (krisdale) [inline source/clear-linux/get-started/virtual-machine-install/vmware-esxi-install-cl.rst]: The removed sentence was a duplicate.

File: source/clear-linux/get-started/virtual-machine-install/vmw-player.rst [hunk 3]
```diff
@@ -71,19 +62,14 @@ Install the VMware Workstation Player hypervisor
...
-   :Start-after: image-types-content:
-
-
 Download the latest |CL| installer ISO
 **************************************
 
 Get the latest |CL| installer ISO image from the `image`_ repository.
 Look for :file:`clear-[version number]-installer.iso.xz`.
 
+Visit :ref:`image-types` for additional information about all available |CL| images.
```
Comment (mvincerx) [inline source/clear-linux/get-started/virtual-machine-install/vmw-player.rst]: At the start of line 56 and 60, please fix the formatting errors. Check that these lines are left-aligned with the line that precedes them.

File: source/clear-linux/get-started/virtual-machine-install/vmware-esxi-install-cl.rst (hunk 2 above)
Comment (mvincerx) [reply source/clear-linux/get-started/virtual-machine-install/vmware-esxi-install-cl.rst]: OK.

File: source/clear-linux/get-started/virtual-machine-install/vmware-esxi-preconfigured-cl-image.rst [hunk 4]
```diff
@@ -39,6 +38,8 @@ this command:
 
    curl -O https://download.clearlinux.org/image/clear-$(curl https://download.clearlinux.org/latest)-vmware.vmdk.xz
 
+Visit :ref:`image-types` for additional information about all available |CL| images.
+
 .. include:: ../../guides/maintenance/download-verify-uncompress-linux.rst
    :Start-after: verify-linux:
    :end-before: To uncompress a GZ
```
Comment (mvincerx) [inline source/clear-linux/get-started/virtual-machine-install/vmware-esxi-preconfigured-cl-image.rst]: At bottom of page, can you please change "See also" to our new "Related Topics"?

File: source/clear-linux/get-started/virtual-machine-install/vmw-player.rst (hunk 3 above)
Comment (krisdale) [reply source/clear-linux/get-started/virtual-machine-install/vmw-player.rst]: Good catch!

**Comment by mvincerx** [comment]
Thanks for going the extra mile on all these files. Great work :)

**Pull request merged**